airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -15,11 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 import tempfile
-from typing import Any, Generator, Optional, Tuple, Union
+from typing import Any, Dict, Generator, Optional, Tuple, Union
 
 import yaml
 from cached_property import cached_property
+from flask_appbuilder.fieldwidgets import BS3TextFieldWidget
+from flask_babel import lazy_gettext
 from kubernetes import client, config, watch
+from wtforms import BooleanField, Field, StringField
```

Comment:
I think it's probably worth delaying these imports until inside `get_connection_form_widgets` -- that way we don't need to import the FAB stack in the worker.

Reply:
Makes sense.

Reply:
I also remove it from the BaseHook.